internal/pkg/cli/env_show_test.go

```diff
@@ -293,27 +293,27 @@ func TestEnvShow_Execute(t *testing.T) {
 
 			wantedContent: `About
 
-  Name              testEnv
-  Production        false
-  Region            us-west-2
-  Account ID        123456789012
+  Name        testEnv
+  Production  false
+  Region      us-west-2
+  Account ID  123456789012
 
 Services
 
-  Name              Type
-  ----              ----
-  testSvc1          load-balanced
-  testSvc2          load-balanced
-  testSvc3          load-balanced
+  Name      Type
+  ----      ----
+  testSvc1  load-balanced
+  testSvc2  load-balanced
+  testSvc3  load-balanced
 
 Tags
 
   Key                  Value
   ---                  -----
   copilot-application  testApp
   copilot-environment  testEnv
-  key1              value1
-  key2              value2
+  key1    value1
+  key2    value2
```

Comment:
it's such a pain adjusting the column width :cool-crying:... I wonder if there is a way for tabwriter to dynamically adjust the width based on max width of a column...